include/picongpu/fields/incidentField/profiles/TWTSPulse.def

```diff
@@ -0,0 +1,170 @@
+/* Copyright 2014-2025 Axel Huebl, Alexander Debus, Richard Pausch, Sergei Bastrakov
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/fields/background/templates/twtstight/TWTSTight.hpp"
+
+namespace picongpu::fields::incidentField
+{
+    namespace profiles
+    {
+        struct window
+        {
+            HDINLINE static float_X switchAt(
+                float_X const currentStep,
+                float_X const switchStartStep,
+                float_X const switchEndStep,
+                float_X const length)
+            {
+                constexpr float_X a0 = 0.3635819;
+                constexpr float_X a1 = 0.4891775;
+                constexpr float_X a2 = 0.1365995;
+                constexpr float_X a3 = 0.0106411;
```

Comment:
What are these constants? Could you add a comment what these magic numbers represent?